contrib/capnproto/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "capnproto"
+pkgver = "1.0.2"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = ["automake", "libtool", "pkgconf"]
+makedepends = ["linux-headers"]
+pkgdesc = "Cap'n Proto serialization/RPC system"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "MIT"
+url = "https://capnproto.org"
+source = f"{url}/capnproto-c++-{pkgver}.tar.gz"
+sha256 = "9057dbc0223366b74bbeca33a05de164a229b0377927f1b7ef3828cdd8cb1d7e"
+
+
+if self.profile().cross:
+    hostmakedepends += ["capnproto", "capnproto-devel"]
+    configure_args = ["--with-external-capnp"]
```

Comment:
```suggestion
    hostmakedepends += ["capnproto-devel"]
    configure_args = ["-DEXTERNAL_CAPNP=ON", "-DBUILD_TESTING=OFF"]
```

after fixing the above this works